src/components/List/Events/List.jsx

```diff
@@ -120,11 +125,23 @@ function Lists(props) {
                 </span>
                 <span className="event-list-description-place">
                   {eventItem?.location?.map((place) => {
-                    return bilingual({
-                      en: place?.name?.en,
-                      fr: place?.name?.fr,
-                      interfaceLanguage: user?.interfaceLanguage?.toLowerCase(),
-                    });
+                    if (!place?.isVirtualLocation)
+                      return bilingual({
+                        en: place?.name?.en,
+                        fr: place?.name?.fr,
+                        interfaceLanguage: user?.interfaceLanguage?.toLowerCase(),
+                      });
+                    else if (place?.isVirtualLocation)
+                      return (
+                        <>
+                          {eventItem?.location?.length > 1 && <>&nbsp;|&nbsp;</>}
```

Comment:
This logic depends on the second location being virtual. What if the first location is virtual and the second is not, then it will give " | first second".